ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,458 +1,470 @@
+{%     if router_bgp.address_family_ipv6 is arista.avd.defined %}
    !
    address-family ipv6
-{%       if router_bgp.address_family_ipv6.peer_groups is defined and router_bgp.address_family_ipv6.peer_groups is not none %}
-{%          for peer_group in router_bgp.address_family_ipv6.peer_groups | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_in is defined and router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_in is not none %}
+{%         for peer_group in router_bgp.address_family_ipv6.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_in is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_out is defined and router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_out is not none %}
+{%             if router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_out is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_ipv6.peer_groups[peer_group].route_map_out }} out
 {%             endif %}
 {%             if router_bgp.address_family_ipv6.peer_groups[peer_group].activate == true %}
       neighbor {{ peer_group }} activate
 {%             elif router_bgp.address_family_ipv6.peer_groups[peer_group].activate == false %}
       no neighbor {{ peer_group }} activate
 {%             endif %}
-{%          endfor %}
-{%       endif %}
-{%       if router_bgp.address_family_ipv6.neighbors is defined and router_bgp.address_family_ipv6.neighbors is not none %}
-{%          for neighbor in router_bgp.address_family_ipv6.neighbors | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_ipv6.neighbors[neighbor].route_map_in is defined and router_bgp.address_family_ipv6.neighbors[neighbor].route_map_in is not none %}
+{%         endfor %}
+{%         for neighbor in router_bgp.address_family_ipv6.neighbors | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_ipv6.neighbors[neighbor].route_map_in is arista.avd.defined %}
       neighbor {{ neighbor }} route-map {{ router_bgp.address_family_ipv6.neighbors[neighbor].route_map_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_ipv6.neighbors[neighbor].route_map_out is defined and router_bgp.address_family_ipv6.neighbors[neighbor].route_map_out is not none %}
+{%             if router_bgp.address_family_ipv6.neighbors[neighbor].route_map_out is arista.avd.defined %}
       neighbor {{ neighbor }} route-map {{ router_bgp.address_family_ipv6.neighbors[neighbor].route_map_out }} out
 {%             endif %}
 {%             if router_bgp.address_family_ipv6.neighbors[neighbor].activate == true %}
```

Comment:
No check if `router_bgp.address_family_ipv6.neighbors[neighbor].activate` is defined